esrally/tracker/tracker.py

```diff
@@ -74,16 +74,19 @@ def create_track(cfg):
     track_name = cfg.opts("track", "track.name")
     indices = cfg.opts("generator", "indices")
     root_path = cfg.opts("generator", "output.path")
-    target_hosts = cfg.opts("client", "hosts")
-    client_options = cfg.opts("client", "options")
+    target_hosts = cfg.opts("client", "hosts").default
+    client_options = cfg.opts("client", "options").default
     data_streams = cfg.opts("generator", "data_streams")
 
-    client = EsClientFactory(
-        hosts=target_hosts.all_hosts[opts.TargetHosts.DEFAULT], client_options=client_options.all_client_options[opts.TargetHosts.DEFAULT]
+    distribution_flavor, distribution_version, _ = factory.cluster_distribution_version(target_hosts, client_options)
+    client = factory.EsClientFactory(
+        hosts=target_hosts,
+        client_options=client_options,
+        distribution_version=distribution_version,
+        distribution_flavor=distribution_flavor,
     ).create()
 
-    info = client.info()
-    console.info(f"Connected to Elasticsearch cluster [{info['name']}] version [{info['version']['number']}].\n", logger=logger)
+    console.info(f"Connected to Elasticsearch cluster version [{distribution_version}]\n", logger=logger)
```

Comment:
Should we log the flavor too?

Reply:
I thought about that, but when you're targeting stateful elasticsearch you just get `default` back, which is a little confusing, so I just left it out entirely. Happy to include it if you think it's important though.

Reply:
Yes I think it's important to have Rally confirm what sort of cluster it thinks it's benchmarking.

Reply:
Sure, addressed in https://github.com/elastic/rally/pull/1738/commits/27f1f1268e3c087eb41cdfa07e2f32aba3640e8b. I've kicked off a long running test benchmark too. Will likely merge early next week.